docs/adguard-for-android/solving-problems/low-level-settings.md

```diff
@@ -108,21 +108,25 @@ Every encrypted Internet connection has an unencrypted part. This is the very fi
 
 #### OCSP checking
 
-Once enabled, this option runs asynchronous OCSP checks to check whether the website’s SSL certificate is revoked.
+This option runs asynchronous OCSP checks to check whether the website’s SSL certificate is revoked.
```

Comment:
If this is enabled, AdGuard will perform asynchronous OCSP status checks to verify that a website's SSL certificate has not been revoked.

or

If this is enabled, AdGuard will perform asynchronous OCSP checks to get the revocation status of a website's SSL certificate.